docs/docs/07-forms.md

```diff
@@ -35,7 +35,7 @@ Like all DOM events, the `onChange` prop is supported on all native components a
 
 ## Controlled Components
 
-An `<input>` with `value` set is a *controlled* component. In a controlled `<input>`, the value of the rendered element will always reflect the `value` prop. For example:
+An `<input>` with a `value` prop is a *controlled* component:
```

Comment:
Keep it simple then:

> A **controlled** `<input>` has `value` property. Rendering a controlled `<input>` will reflect the value of the `value` prop.

It does not sound sound natural, but is succinct. The present description is too verbose. Furthermore, this definition gives a hint that it reflects the value of the property at the time of each render.